pkg/gui/presentation/branches.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 var branchPrefixColorCache = make(map[string]style.TextStyle)
+var goneBranchText = "↑gone"
 
 func GetBranchListDisplayStrings(branches []*models.Branch, fullDescription bool, diffName string) [][]string {
```

Comment:
rather than use a `goneBranchText` variable I'd instead pass in `gui.Tr` into this function. We really should wrap this file's functions into a struct that has a Tr field so that we don't need to pass that around through all the functions, but it's okay if you don't want to go that far.

Reply:
Yeah, I went with passing `gui.Common.Tr`, because like you said I don't want to go that far for this PR. I feel like refactoring should be separated to this, especially since it might not be a small change.